### ISSUE TITLE
do a major service on Gerald

### ISSUE THREAD

**Issue by DefProc** [DoESLiverpool/somebody-should#518]
Gerald has been collecting enough fixes, and has enough worn parts that it's worth taking it ***out of service for a week or more*** to perform a major service and get our large cutter running more reliably again.

We'll need to collate a list so there's a clear idea of all the things than need doing as part of a service, where appropriate, these should link back to an issue (open or closed):

 * [x] replace the lower bed toothed belt(s) #288 
 * [ ] replace the screw, refit and loctite the bearing cover on the y-axis slides #287 #429 (requires disassembly of the y-axis linear bearing assembly)
 * [ ] loctite the bearing plate mounting screws on reassembly #517 
 * [x] fit the safety cut-out switch more securely #516 
 * [x] new mirrors and lens for gerald?
 * [x] clean down the whole inside of cutting residue,
 * [x] fit a 15mm drive belt to the x-axis #501 
 * [x] build a permanently vented exhaust setup #474 
 * [ ] replace the short motor initial drive belts
 * [x] refit the front panel lock
 * [x] replace the control panel for one that's not peeling off
 * [x] fit new o-rings behind the intermediate mirrors #387 
 * [ ] Any more?

It's already looks like a big job, and some of these require more than one person. It would be great to get a working group together from the wider community. Can you help with this service?

**Comment by amcewen** [comment]
Organisers' meeting notes: @jackie1050 and @huffeec have offered to help.  John Cain possibly too.

All to suggest any further things that need doing.

**Comment by stevesparrow** [comment]
I'm conscious that Patrick, you're off for most of August, and we want to give people plenty of notice for a refurb is the laser is going to be down.

Should we try and set a date for this even if it's in September, so when people are frustrated they know there is  a service coming?

Thanks, Steve

_Sent from my Asus P027 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by amcewen** [comment]
From the organisers meeting, @DefProc what's the status on this?

**Comment by DefProc** [comment]
Happy to take a week from September, how about the 25th?

Need to order a lower toothed belt from China #288 (which hasn't happened yet) as HPC was unable to supply the belt. I think all the other parts can come from the UK (eg ducting, mirrors, lenses and o-rings) or we already have in the laser cutter cupboard (the screw that came out, upper belts)

**Comment by DoESsean** [comment]
Is that the belt that we were waiting ages on?

Happy to order a new anything if someone lets me know what it is we need.
There are a few suggestions on the issue page @defproc has linked, and I
know I'd end up picking the wrong one...

**Comment by DoESsean** [comment]
No-one has said anything about the belt. As I understand, this is still an issue.

**Comment by goatchurchprime** [comment]
According to Karl D. the right hand side of the bed is visibly lower and needs 2cm of packing under the stock to level it.

**Comment by jackie1050** [comment]
There's a screw thread at the back right that let's you adjust the bed height - lasts a while

On 14 Mar 2018 17:52, Julian Todd <notifications@github.com> wrote:

According to Karl D. the right hand side of the bed is visibly lower and needs 2cm of packing under the stock to level it.

**Comment by jackie1050** [comment]
Not sure which issue to add this to ...
@zarino and I were trying to cut some ply yesterday (parts of the crate Martin's CNC came in ).
This was approx. 5mm thickness. Settings : speed 5 power 70 corner power 65. It cut ok in top left corner but even after 4 cuts it didn't cut through in other places (more than 300mm from top left).

**Comment by Sean-anotherone** [comment]
I have proposed elsewhere that gerald needs a proper overhaul, the list at the start of this thread seems fairly comprehensive - although a couple of those things have already been done.

The belts should not be a problem at all - https://www.beltingonline.com/ have always had every belt/pulley I have required ex-stock, and they are in Hampshire - more than new belt is required though - there are mechanical issues under the bed, too, and even properly diagnosing them will require a stripdown.

Mike and I have an action to get motivated and relocate Morris into the laser room so he can be worked on and brought into service - once that is complete I suggest taking Gerald offline for as long as it takes to fix everything.
